<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>351</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iconButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Icons</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="hiddenButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="renameButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mkDirButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>MkDir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RM</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
